src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -445,9 +445,9 @@ We also have several older or less common API key type. To rotate these keys:
         title="Rotate REST API key"
     >
 
-      This is an older key that is not recommended for use by customers, but is still supported by New Relic. We now recommend using the user key instead of the REST API key.
+      This is an older key that is deprecated, but is still supported by New Relic. We now recommend using the user key instead of the REST API key.
```

Comment:
```suggestion
      This is an older key that's deprecated but still supported by New Relic. We now recommend using the user key instead of the REST API key.
```